services/cronback-project-srv/db_model/projects.rs

```diff
@@ -0,0 +1,57 @@
+//! `SeaORM` Entity. Generated by sea-orm-codegen 0.11.3
+
+use chrono::{DateTime, Utc};
+use dto::{FromProto, IntoProto};
+use lib::model::ValidShardedId;
+use lib::types::ProjectId;
+use sea_orm::entity::prelude::*;
+
+#[derive(Clone, Debug, PartialEq, DeriveEntityModel, Eq)]
+#[sea_orm(table_name = "projects")]
+pub struct Model {
+    #[sea_orm(primary_key, auto_increment = false)]
+    pub id: ValidShardedId<ProjectId>,
+    pub created_at: DateTime<Utc>,
+    pub last_status_changed_at: DateTime<Utc>,
```

Comment:
Maybe `status_updated_at`?

I'm not sure we will get enough value by having this instead of just `updated_at`. Possibly, looking at project status update events log will be the best place to find out what happened and when. What do you think?